src/index.js

```diff
@@ -205,18 +206,22 @@ const processResults = ({ summaries, errors }) => {
       error,
     };
   } else {
+    let extra_data = {};
     return {
-      summary: summaries
-        .map(({ path, url, summary }) => {
+      summary: data
+        .map(({ path, url, summary, results }) => {
           if (path) {
+            extra_data = { path, summary, reportHtml: results.report };
             return `Summary for directory '${chalk.magenta(path)}': ${summary}`;
           }
           if (url) {
+            extra_data = { url, summary, reportHtml: results.report };
             return `Summary for url '${chalk.magenta(url)}': ${summary}`;
           }
           return `${summary}`;
```

Comment:
I think we still get report HTML at this point? We may need `extra_data = { summary, reportHtml: results.report };` before the final return.

Reply:
If so, maybe you could set `extra_data = { summary, reportHtml: results.report };` at the start of the map, then add `extra_data.path = path` etc in the `if`s to reduce some duplication?